app/models/ems_event.rb

```diff
@@ -196,6 +196,7 @@ def get_target(target_type)
     target_type = "src_vm_or_template"  if target_type == "src_vm"
     target_type = "dest_vm_or_template" if target_type == "dest_vm"
     target_type = "middleware_server"   if event.event_type == "hawkular_alert"
+    target_type = "container_node"      if event.event_type == "datawarehouse_alert"
```

Comment:
I already dislike the one line above, where it does implicitly connects `hawkular` to `middleware_server`
Is it possible to switch on something else than `datawarehose_alert` ? Maybe not switch on `event.event_type` but on some other meta data?